webapp-django/crashstats/topcrashers/tests/test_views.py

```diff
@@ -271,12 +273,6 @@ def test_with_unsupported_product(self):
         assert response.status_code == 404
 
     def test_without_any_signatures(self):
-        url = self.base_url + '?product=WaterWolf&version=19.0'
-        response = self.client.get(self.base_url, {
-            'product': 'WaterWolf',
-        })
-        assert url in response['Location']
```

Comment:
This seems to test "does it redirect" which is tested elsewhere so we don't need to do it again.